assets/css/sass/utils/_mixins.scss

```diff
@@ -49,3 +49,13 @@
 	border-radius: 0;
 	box-shadow: inset 0 -1px 0 rgba( #000, 0.3 );
 }
+
+@mixin wrapBreakWord {
+	// https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap
+	// https://developer.mozilla.org/en-US/docs/Web/CSS/word-break
+	// This is the current standard, works in most browsers.
+	overflow-wrap: anywhere;
```

Comment:
```suggestion
	overflow-wrap: anywhere;
	// IE11 doesn't support overflow-wrap neither word-break: break-word, so we fallback to work-break: break-all.
	word-break: break-all;
```

Reply:
Agreed - I'll test in IE today and use that or `word-wrap` alias, which looks like it might work in IE ([was previously an MS extension](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap)). Incidentally that what was used previously for the `table.woocommerce-checkout-review-order-table` case, so probably not a dangerous change :)

Reply:
`-ms-word-break` did the trick. 21fa05f49cf3b377db3c2fe3f20076f49f66725b